tests/test_pipeline/test_hierarchical_pipeline.py

```diff
@@ -412,9 +412,9 @@ def test_interval_metrics(product_level_constant_hierarchical_ts, metric_type, r
     results, _, _ = pipeline.backtest(
         ts=ts,
         metrics=[metric],
-        n_folds=2,
+        n_folds=1,
```

Comment:
Why did you make changes to this test?

Reply:
With old parameters with `horizon=1` and `n_folds=1` (for `forecast_params`) we have only 1 point to infer interval width. It now leads to error, because it is too small amount of points.

With new parameters we have 2 points already and it is fine. I was forced to make `n_folds=1` for `backtest` because there are very few points: only 4. It is impossible to make 2 folds for backtest and 2 folds for interval estimation in that case.